pkg/strategy/sentinel/math.go

```diff
@@ -0,0 +1,28 @@
+package sentinel
+
+import "math"
+
+func mean(floats []float64) float64 {
```

Comment:
I think we have implemented the "mean" somewhere for float64 slice, you can grep

Reply:
replaced by floats.Slice